f8a_worker/defaults.py

```diff
@@ -155,5 +155,26 @@ def select_random_github_token(cls):
             raise F8AConfigurationException("Limit for unauthorized GitHub access exceeded.")
         return token, headers
 
+    @classmethod
+    def libraries_io_project_url(cls, ecosystem, name):
+        """Construct url to endpoint, which gets information about a project and it's versions."""
+        url = '{api}/{platform}/{name}'. \
+            format(api=cls.LIBRARIES_IO_API,
+                   platform=ecosystem,
+                   name=name)
+
+        if not cls.LIBRARIES_IO_TOKEN or cls.LIBRARIES_IO_TOKEN == 'not-set':
+            raise F8AConfigurationException("LIBRARIES_IO_TOKEN has not been set.")
+
+        # 'no-token' value forces the API call to not use ANY token.
+        # It works, but if abused, they can ban your IP, so use with caution.
+        if cls.LIBRARIES_IO_TOKEN != 'no-token':
```

Comment:
Is this condition really required? I believe that `not-set` and `no-token` are same, so this is redundant.

Reply:
Yes, it is. See above comment.